src/__tests__/createManager.test.js

```diff
@@ -165,7 +166,8 @@ describe('createManager', function() {
     var mockOnSelection = jest.fn();
     var manager = createManagerWithMockedElements({
       onSelection: mockOnSelection,
-      closeOnSelection: false
+      closeOnSelection: false,
+      closeOnBlur: false
     });
     manager.closeMenu = jest.fn();
     manager.handleSelection('foo', { bar: 1 });
```

Comment:
Can you please add some more tests validating the effects on Button and Menu of turning this *off*?

Reply:
Hi @davidtheclark ! Let me know if more tests are required. Not sure if these are what you were expecting.

Reply:
Sorry for the delay :(